### ISSUE TITLE
class ETM self.kernels = kernels.detach().clone()

### ISSUE THREAD

**Issue by wxk2024** [jiawen-zhu/AsymTrack#7]
我发现训练过程中有这一行代码，这不就代表着梯度被截断了，也就没办法学习了。希望大佬解释一下，谢谢
